Daytime-Operations/Auxiliary-Telescope/AT-Venting/Prepare-for-Vent.rst

```diff
@@ -0,0 +1,137 @@
+  
+  No earlier than 2.5 hours before sunset and no later than 1 hour before sunset, :ref:`prepare for on-sky operations <AT-On-sky-WEP>`. 
+
+.. _Prepare-for-vent-Procedure-Steps:
+
+Procedure Steps
+===============
+#. Review procedure :ref:`prerequisites <Prepare-for-vent-Prerequisites>` and confirm they have been executed. 
+
+#. Announce in the #summit-announce and #summit-auxtel Slack channel that AuxTel is going to vent, and telescope and dome will move. 
+
+#. Command the telescope and dome to prepare for venting. 
+   Load the ``auxtel/preparefor/vent.py`` script from the LOVE ``ATQueue`` panel, under ``AVAILABLE SCRIPTS`` by clicking on the blue icon. 
+
+   .. note::
+     The ``auxtel/preparefor/vent.py`` will only run when the Sun's elevation is between 55 deg and 5 deg above horizon. 
+     Otherwise it will give you a warning and fail. 
+     
+   In the ``configuring script:vent`` screen that pops up, leave the ``CONFIG`` box empty. 
+
+   .. figure:: ./_static/PrepareforVent_AuxTel.png
+     :name: prepareforonsky_AuxTel
+    
+     Screenshot of LOVE launching ``auxtel/preparefor/vent.py`` script. 
+
+   The script performs the following steps:
+       - Check that all components are enabled. 
+       - Disable dome following. 
+       - Home the dome. 
+       - Disable AOS open loop corrections. 
+       - Close M1 primary mirror cover. 
+       - Close dome, if it was open. 
+       - Enable AOS open loop correction.
+       - Point telescope horizontally; az = 180, el = 30, rot = 0 deg. 
+       - Disable AOS open loop corrections. 
+       - Slew dome to face opposite the Sun; az = Sun's azimuth - 180 deg.
+
+   .. note::
+     The ``auxtel/preparefor/vent.py`` script will keep running until the Sun's elevation is 5 degrees above the horizon, 
+     repositioning the dome azimuth position every minute. 
+     
+   .. figure:: ./_static/PrepareforVent_AuxTel_running.png
+     :name: script prepareforonsky_AuxTel running
+
+     ``auxtel/preparefor/vent.py`` script running until observer stops it or the Sun reaches 5 deg above horizon. 
+
+#. Proceed to the AuxTel dome, and open the dome shutter manually two thirds of the way using the local buttoms at the top of the stairs. 
+   The dome main shutter should stay around one third close to avoid too much Sun's light hitting on the dome. 
+
+#. If the wind speed is below 8 m/s, manually open vent gate #3 using the remote controller and turn on the extraction fan power to max.  
+   If wind speed is above or close to 8 m/s, keep vent gates and extraction fan off. 
+
+   .. figure:: ./_static/PrepareforVent_AuxTel_VentGate3andFan.png
+      :name: Dome Vent Gate 3 and Extraction Fan 
+
+      AuxTel dome vent gate #3 and extraction fan with its controller located at the dome pier. 
+
+#. Visually confirm in LOVE that the system is venting. 
+        - Telescope is closer to horizontal pointing; az = 180, el = 30 deg, rot = 0. 
+        - M1 Mirror cover is closed. 
```

Comment:
But on the image (Daytime-Operations/Auxiliary-Telescope/AT-Venting/_static/PrepareforVent_AuxTel.png) , the M1 Mirror cover is opened

Reply:
Yes, sorry,  I wrongly uploaded the prepare for on-sky screenshot. Solved that now.